source/Jacwright/RestServer/RestServer.php

```diff
@@ -358,7 +358,9 @@ public function getMethod()
 			$method = 'PUT';
 		} elseif ($method == 'POST' && strtoupper($override) == 'DELETE') {
 			$method = 'DELETE';
-		}
+		} elseif ($method == 'PATCH' && strtoupper($override) == 'PATCH') {
+            $method = 'PATCH';
+        }

```

Comment:
Ah, yes, of course. If the other overrides are there, this might as well, too. Got it.